WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -213,6 +213,12 @@
         sign_in organizer
       end
 
+      it 'cannot confirm competition' do
+        patch :update, id: competition, competition: { name: competition.name }, commit: "Confirm"
```

Comment:
Argh, looks like it requires IE10: http://www.w3schools.com/tags/att_input_formaction.asp

Adding JS is probably not worth it.